### PR TITLE
Category page

### PR DESCRIPTION
**Pull request by Parkuel**
### Description

Please provide a brief description of the changes you have made in your PR.

### Type of change

- [x] Feature (introduces new functionality)
- [ ] Bugfix
- [ ] Configuration
- [ ] Refactoring (doesn't change functionality, only the code)
- [ ] Style changes (doesn't affect functionality, only the look of the components)

### PR Checklist

- [x] Synced up with latest from the main branch/Code is linted
- [x] Description of the changes has been added (Screenshots/Video)
- [x] Chakra components/constants/hooks have been used wherever possible

> If not explain here

- [x] Commit messages follow [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)
- [x] Console has been checked for warnings

#### Screenshots/Video before change
catergoryBefore

Add screenshots from the [current deployed state](https://sandbox-money-app.herokuapp.com/) of the changed components here.

#### Screenshots/Video after change

Add screenshots with the changes app
catergoryAfter
lied here.

### CODE REVIEW

File: src/pages/Categories.jsx [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+      <Grid templateColumns="repeat(2, 1fr)" height="100vh">
+        <GridItem className="expenseCol">
+          <AccordionHeadings
+            headingOne="Categories of Expenses "
+            headingTwo="Add expense Category"
+          />
+          <AccordionComponent name="Category 1" message="I am random message" />
+          <AccordionComponent name="Category 1" message="I am random message" />
+          <AccordionComponent name="Category 1" message="I am random message" />
+        </GridItem>
```
Comment (PavelTitov16) [inline src/pages/Categories.jsx:17]: maybe it's better to create like array maping to create categories components?

Comment (Parkuel) [reply src/pages/Categories.jsx:17]: Yea 😅. Funny, i did ask about arrays today. You're right, I'll implement that

File: src/components/AccordionComponent/AccordionComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+              <Box flex="2" px={10} textAlign="left">
+                <p>{props.name}</p>
+              </Box>
+
+              <Box flex="2" px={5} textAlign="right">
+                <Button>Edit</Button>
+              </Box>
+
+              <Box textAlign="right">
+                <Button variant="danger">Delete</Button>
```
Comment (kalashnikau) [inline src/components/AccordionComponent/AccordionComponent.jsx:40]: text need to be localised (i18n)

File: src/components/AccordionComponent/AccordionComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                  borderRadius="50%"
+                ></Button>
+              </Box>
+
+              <Box flex="2" px={10} textAlign="left">
+                <p>{props.name}</p>
+              </Box>
+
+              <Box flex="2" px={5} textAlign="right">
+                <Button>Edit</Button>
```
Comment (kalashnikau) [inline src/components/AccordionComponent/AccordionComponent.jsx:36]: text need to be localised (i18n)

File: src/components/Footer/Footer.jsx [hunk 4]
```diff
@@ -1,9 +1,25 @@
...
+      bottom="0"
+      height="10vh"
+      boxShadow="lg"
+      justify="center"
+      marginTop="20px"
+      zIndex="-1"
+    >
+      <footer>
+        <Text color="#ccb" fontSize="xl">
+          Exadel Internship {currentYear}
```
Comment (kalashnikau) [inline src/components/Footer/Footer.jsx:20]: text need to be localised (i18n)

File: src/pages/Categories.jsx [hunk 5]
```diff
@@ -1,3 +1,31 @@
...
+
 export const Categories = () => {
-  return <></>;
+  return (
+    <>
+      <Grid templateColumns="repeat(2, 1fr)" height="100vh">
+        <GridItem className="expenseCol">
+          <AccordionHeadings
+            headingOne="Categories of Expenses "
+            headingTwo="Add expense Category"
```
Comment (kalashnikau) [inline src/pages/Categories.jsx:12]: text need to be localised (i18n)

**Comment by Parkuel** [comment]
Blockers
- I could not use Vstacks because of the dropdown requirements, hence why I am using accordions.
- The suggested 'IconButton' also generated the same button descendant error.
- The current approach where 'Edit' and 'Delete' buttons are placed outside 'AccordionButton' and then repositioned is not responsive.

What I want to do next
- Accordion might have to be built from scratch

If after my journey it is not done, I will recheck it and make sure I get it to work. Apologies for the extra work I might be giving my colleagues. I will pick up the pace. Thanks

File: src/components/AccordionHeadings/AccordionHeadings.jsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import { AddIcon } from '@chakra-ui/icons';
+import { Box, useColorModeValue } from '@chakra-ui/react';
+
+export const AccordionHeadings = (props) => {
+  const bgMain = useColorModeValue('orange.50', 'teal.800');
```
Comment (kalashnikau) [inline src/components/AccordionHeadings/AccordionHeadings.jsx:5]: use that variable from theme hook

**Review by kalashnikau — APPROVED** [review]

**Review by Yulatun — APPROVED** [review]

**Review by itsnotpolza — APPROVED** [review]

**Pull request merged**